### PR TITLE
Release 2.0.0

### PR DESCRIPTION
**Pull request by Baggerone**
Modify to work with simplesamlphp 1.15.2

### CODE REVIEW

File: lib/Auth/Process/AddIdp2NameId.php [hunk 1]
```diff
@@ -92,29 +91,17 @@ public function __construct($config, $reserved) {
     }
 
     /**
-     * @param $value array|string The NameID string or array (with a Value entry)
+     * @param $nameId \SAML2\XML\saml\NameID
      * @param $IDPNamespace string
-     * @return array|string
-     * @throws SimpleSAML_Error_Exception
+     *
+     * Modifies the nameID object but adding text to the end of its value attribute
```
Comment (forevermatt) [inline lib/Auth/Process/AddIdp2NameId.php]: Small typo: `but adding` --> `by adding`

File: lib/Auth/Source/SP.php [hunk 2]
```diff
@@ -131,6 +132,7 @@ public function getIdPMetadata($entityId) {
 			return $metadataHandler->getMetaDataConfig($entityId, 'shib13-idp-remote');
 		} catch (Exception $e) {
 			/* Metadata wasn't found. */
+            SimpleSAML\Logger::debug('getIdpMetadata: ' . $e->getMessage());
```
Comment (forevermatt) [inline lib/Auth/Source/SP.php:135]: Do you want to make the message logged for each of these distinct from each other in some way? I'm not sure how important that is, but it seemed worth suggesting.

File: www/metadata.php [hunk 3]
```diff
@@ -5,6 +5,12 @@
 
 require_once('../www/_include.php');
 
+use SAML2\Constants;
+use SimpleSAML\Utils\Auth as Auth;
```
Comment (forevermatt) [inline www/metadata.php:9]: Not that it changes much, by the `as _____` isn't necessary when the text you provide after `as` exactly matches the class name. In other words, this...
```php
use SimpleSAML\Utils\Auth as Auth;
```
... is the same as this:
```php
use SimpleSAML\Utils\Auth;
```

File: www/sp/saml2-acs.php [hunk 4]
```diff
@@ -1,7 +1,9 @@
 <?php
-
+ini_set('display_errors', 1);
```
Comment (forevermatt) [inline www/sp/saml2-acs.php]: Whoops, looks like this was accidentally left in here.

**Review by forevermatt — CHANGES_REQUESTED** [review]
Okay, I've finished a pass through reviewing these changes. The one critical thing to change is removing the `ini_set('display_errors', 1);` line. My other comments are just small things, not blockers.

**Review by fillup — APPROVED** [review]

**Pull request merged**